src/common-components/LargeScreenRightLayout.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Col } from '@edx/paragon';
+
+const LargeScreenRightLayout = (props) => {
+  const { children } = props;
+
+  return (
+    <Col xs={6} className="vh-100 pt-5">
```

Comment:
Remove `vh-100` class, it's causing problems when the form changes its size on success/error messages.